src/frontend/web_application/src/scenes/Signin/components/SigninForm/presenter.jsx

```diff
@@ -37,10 +37,25 @@ class SigninForm extends Component {
     this.setState(generateStateFromProps(this.props));
   }
 
+  componentDidMount() {
+    setTimeout(this.setValues(), 1);
+  }
+
   componentWillReceiveProps(newProps) {
     this.setState(generateStateFromProps(newProps));
   }
 
+  setValues = () => {
+    const password = this.passwordInputRef.value;
+    const username = this.usernameInputRef.value;
+    this.setState(prevState => ({
+      formValues: {
+        password: prevState.password !== password && password,
```

Comment:
in case prevState.password & password are equal, this will evaluate `setState{{formValues: { password: false ... } })` which is incorrect.

this should be good:

```
this.setState(prevState => ({
  formValues: {
    password,
...
}));
```

Reply:
yes of course /o\